lib/tasks/sumofus.rake

```diff
@@ -0,0 +1,51 @@
+require 'open-uri'
+
+namespace :sumofus do
+  desc 'Import legacy actions into your database from a specified file'
+  task seed_legacy_actions: :environment do
+    if ARGV[1].nil?
+      abort('Requires a valid url to a file containing the legacy actions to seed.')
+    end
+
+    if ARGV[2].nil?
+      abort('Requires a valid url to a file containing a default header image to attach to the page.')
+    end
+
+    file_handle = open(ARGV[1])
+    data = file_handle.read
+    file_handle.close
+
+    image_handle = open(ARGV[2])
+
+    parsed_data = JSON.load data
+    count = 0
+    layout_id = LiquidLayout.where(title: 'Petition With Small Image').first.id
+    parsed_data.each_pair do |k, entry|
+      count +=1
+      title = entry['title'].blank? ? entry['petition_ask'] : entry['title']
+      page = Page.find_or_create_by!(title: title, liquid_layout_id: layout_id)
+      page.content = entry['page_content'].gsub(/(?:\n\r?|\r\n?)/, '<br>')
```

Comment:
The rails console says there aren't any `<br>` tags saved in that particular page's content, so it looks like they're getting eaten along the way. It also tells me there are 423 pages (of the 446) which contain `<br>` tags so given the relative lack of value for any one page, I'm not at all concerned with trying to debug where those tags are going.